readthedocs/core/utils/__init__.py

```diff
@@ -34,12 +40,18 @@ def broadcast(type, task, args, kwargs=None):  # pylint: disable=redefined-built
         servers = getattr(settings, "MULTIPLE_APP_SERVERS", [default_queue])
     elif type in ['build']:
         servers = getattr(settings, "MULTIPLE_BUILD_SERVERS", [default_queue])
+
+    tasks = []
     for server in servers:
-        task.apply_async(
-            queue=server,
-            args=args,
-            kwargs=kwargs,
-        )
+        task_sig = task.s(*args, **kwargs).set(queue=server)
+        tasks.append(task_sig)
+    if callback:
+        task_promise = chord(tasks)(callback).get()
+        log.debug('Sent Chord: {}'.format(locals()))
```

Comment:
If we can't guarantee that `locals()` won't leak things we don't want logged to logging, I'd say this should instead be logging some interpretation of the task name and what was passed in. Perhaps task name + kwarg names?

Reply:
I'll just kill this, as celery with debug logging will show the tasks in console